docs/easy-deployments.md

```diff
@@ -0,0 +1,38 @@
+# Easy Deployments
+
+There are several easy deployments instructions described in this document to help you deploy your fork of the ott-web-app to some of the popular web application hosting platforms.
+
+Currently supported platforms:
+
+- [Github Pages](#github-pages). It's a free web hosting service offered by Github. The main Github Pages advantages are that it's free of charge and easy to use.
+
+Before you follow any of ://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages
```

Comment:
Maybe you should have this suggestions at the top of this doc (since I imagine it'll apply to all of the supported. platforms we will add). Like a "before you try any of these easy deployments, first follow the instructions in docs/build-from-source.md to build your app locally.